app/javascript/controllers/content_list_controller.js

```diff
@@ -2,10 +2,36 @@ import { Controller } from "@hotwired/stimulus"
 import Thumbnail from 'src/modules/thumbnail'
 
 export default class extends Controller {
-  // This listens for a thumbnails-found event. Then it draws each on the page.
-  draw(evt) {
+  // This listens for a thumbnails-found event. Then it draws the media thumbnails in the contents panel
+  drawThumbnails(evt) {
     const thumbnails = evt.detail.
       map((thumbnailData, index) => new Thumbnail(thumbnailData).build(index))
     this.element.innerHTML = thumbnails.join('')
   }
+
+  // Show a different media file when the user selects it in the list
+  // Sends an event that tells the MediaWrapperController to display the selected media.
+  showMedia(evt) {
+    evt.preventDefault();
+    const event = new CustomEvent('thumbnail-clicked', { detail: { index: evt.params.index } })
+    window.dispatchEvent(event)
+
+    const tabs = this.element.closest('[role="tablist"]').querySelectorAll('[role="tab"]')
+    tabs.forEach((target) => {
+      target.classList.remove('active')
+      target.setAttribute("aria-selected", false)
+    })
+
+    const target = this.element
+    target.classList.add('active')
+    target.setAttribute("aria-selected", true)
+  }
```

Comment:
Moved from `app/javascript/controllers/thumbnail_controller.js` controller and renamed for consistency.  Updated references in the code to ensure it works.